30-remote-site-setup.sh

```diff
@@ -170,4 +186,3 @@ else
 fi
 
 set $olde
-
```

Comment:
Not a fan of a comfort newline at eof, ey?

Reply:
Oops, that snuck in there by mistake!